empire/server/core/config.py

```diff
@@ -99,17 +101,30 @@ class EmpireConfig(EmpireBaseModel):
         alias="supress-self-cert-warning", default=True
     )
     api: ApiConfig | None = ApiConfig()
-    starkiller: StarkillerConfig
-    submodules: SubmodulesConfig
-    database: DatabaseConfig
+    starkiller: StarkillerConfig = StarkillerConfig()
+    submodules: SubmodulesConfig = SubmodulesConfig()
+    database: DatabaseConfig = DatabaseConfig(
+        sqlite=SQLiteDatabaseConfig(),
+        mysql=MySQLDatabaseConfig(),
+        defaults=DatabaseDefaultsConfig()
+    )
     plugins: dict[str, dict[str, str]] = {}
-    directories: DirectoriesConfig
-    logging: LoggingConfig
-    debug: DebugConfig
+    directories: DirectoriesConfig = DirectoriesConfig(
+        downloads=Path("empire/server/downloads"),
+        module_source=Path("empire/server/modules"),
+        obfuscated_module_source=Path("empire/server/modules_obfuscated")
```

Comment:
I guess you mean
```python
    module_source: Path = Path("empire/server/modules")
    obfuscated_module_source: Path = Path("empire/server/data/obfuscated_module_source")
```
instead of
```python
    module_source: Path("empire/server/modules")
    obfuscated_module_source: Path("empire/server/data/obfuscated_module_source")
```

Reply:
Now that you have the defaults there, you can set `directories: DirectoriesConfig = DirectoriesConfig()`